app/helpers/hosts_helper.rb

```diff
@@ -270,6 +278,17 @@ def overview_fields(host)
     fields
   end
 
+  def host_detailed_status_list(host)
+    host.host_statuses.sort_by(&:type).map do |status|
+      next unless status.relevant?
+      [
+        _(status.name),
+        content_tag(:i, '&nbsp'.html_safe, :class => host_global_status_icon_class(status.to_global)) +
```

Comment:
I think this is adding an unnatural amount of spacing between the icon and status text?